ads/vendors/dable.js

```diff
@@ -52,6 +56,10 @@ export function dable(global, data) {
     widgetOpts.category3 = articleSection3;
     logOpts.category3 = articleSection3;
   }
+  if (orgServiceId) {
+    widgetOpts.orgServiceId = orgServiceId;
```

Comment:
@powerivq these don't get obfuscated?

Reply:
I think it's up to Dable whether to do something about these parameters.

Reply:
@erwinmombay Thank you for the review 🙏 
It's an internally agreed value, so I don't think it'll be a problem but please let me know if you have any concerns.